marlowe-apps/Finder.md

```diff
@@ -30,16 +30,12 @@ marlowe-finder : find active Marlowe contracts
 
 Usage: marlowe-finder [--chain-seek-host HOST_NAME]
                       [--chain-seek-command-port PORT_NUMBER]
-                      [--chain-seek-query-port PORT_NUMBER]
                       [--chain-seek-sync-port PORT_NUMBER]
-                      [--history-host HOST_NAME]
-                      [--history-command-port PORT_NUMBER]
-                      [--history-query-port PORT_NUMBER]
-                      [--history-sync-port PORT_NUMBER]
-                      [--discovery-host HOST_NAME]
-                      [--discovery-query-port PORT_NUMBER]
-                      [--discovery-sync-port PORT_NUMBER] [--tx-host HOST_NAME]
-                      [--tx-command-port PORT_NUMBER]
+                      [--marlowe-sync-host HOST_NAME]
+                      [--marlowe-sync-sync-port PORT_NUMBER]
```

Comment:
Oof, that's an awkward parameter name... but so is `marlowe-sync-marlowe-sync-port`... Are there better alternatives?

how about

```
--sync-host
--sync-marlowe-sync-port
--sync-header-sync-port
--sync-query-port
```

Reply:
The name `marlowe-history` is already taken, but that is most logical.

Reply:
I don't have a preference, but we should make the name uniform across `exe:marlowe-webserver`, `lib:marlowe-apps`, etc.

Reply:
Let's use
```
--sync-host
--sync-marlowe-sync-port
--sync-header-sync-port
--sync-query-port
```
since `sync-sync` is just too weird.

Reply:
Fixed in dd7cbd5b7646c2d03f5ee88274dab8653a21aba8.